wazo_websocketd/bin/init_amqp.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
+# Copyright 2019 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import argparse
+import sys
+
+import kombu
+
+ROUTING_KEYS = [
+    'applications.#',
+    'authorizations.#',
+    'call_log.#',
+    'calls.#',
+    'chatd.#',
+    'collectd.#',
+    'conferences.#',
+    'config.#',
+    'directory.#',
+    'events.#',
```

Comment:
`events.#` is an ACL, not a routing key, no?

Reply:
It was from an example in the documentation. I fixed the doc example to use a working ACL :)